### DISCUSSION TITLE
cut-on-fcp/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#186] [Website Discussions]
# Cut on Final Cut Pro

This page is dedicated to listing all the amazing projects cut on Final Cut Pro.

https://fcp.cafe/cut-on-fcp/

**Comment by florian-duffe** [comment]
Hi Chris, I think I made a mistake with the TV movies. Since FCP dropped the X after Nov 2020 I only wrote {{fcp}} in the box. This seems to not show the box at all - or it's wrong syntax by me.

**Reply by latenitefilms** [reply]
Fixed! It should have been `{{fcpx}}`.

**Comment by florian-duffe** [comment]
ah, thanks. what I meant is: technically it's not correct to write FCPX with projects after 2020 - is there a way to create a new button that says "Final Cut Pro"?

**Reply by latenitefilms** [reply]
Until they get to v11, Final Cut Pro X is still totally valid. Apple still uses Final Cut Pro X in a lot of it's websites. I prefer FCPX, as it's clear.

**Reply by florian-duffe** [reply]
I see your point. 👍🏻
